### DISCUSSION TITLE
Extracting partition blocks locations and dimensions

### DISCUSSION THREAD

**Discussion by DarrenR96** [IENT/YUView#525] [Q&A]
Hi, 
There's a pretty neat visualizer in YUView that allows you to see the partition sizes of HEVC encoded videos. 
Is there any way command line option to extract this data to some json/csv format?

**Comment by ChristianFeldmann** [comment]
Hi. Sorry no that is currently not supported. You could easily add this thoug. Or just use the libde265 project with the interface for it and add it there.
